components/new_channel_modal/new_channel_modal.jsx

```diff
@@ -310,7 +312,10 @@ export default class NewChannelModal extends React.PureComponent {
                     >
                         <Modal.Body>
                             <div className='form-group'>
-                                <label className='col-sm-3 form__label control-label'>
+                                <label
+                                    className='col-sm-3 form__label control-label'
+                                    id='channel_modal_type_label'
```

Comment:
Please change into camelCase.  We have this style guide when writing E2E - https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs